diff --git a/src/index.ts b/src/index.ts
index da4881fa3ec784f13f62cd71b05e2e65fdfc51dd..41c08d45cd788fdb373da5f1ece3ce2d4add5911 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -313,7 +313,7 @@ export class Agent<Env, State = unknown> extends Server<Env> {
    */
   static options = {
     /** Whether the Agent should hibernate when inactive */
-    hibernate: true, // default to hibernate
+    hibernate: false, // default to hibernate
   };
 
   /**
@@ -831,9 +831,9 @@ export class Agent<Env, State = unknown> extends Server<Env> {
   private async _scheduleNextAlarm() {
     // Find the next schedule that needs to be executed
     const result = this.sql`
-      SELECT time FROM cf_agents_schedules 
+      SELECT time FROM cf_agents_schedules
       WHERE time > ${Math.floor(Date.now() / 1000)}
-      ORDER BY time ASC 
+      ORDER BY time ASC
       LIMIT 1
     `;
     if (!result) return;
